### PR TITLE
Function: index on label and list or count vertices/edges

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md [hunk 1]
```diff
@@ -1,27 +1,56 @@
 # CREATE INDEX
 
+Use `CREATE INDEX` to add indexes to existing tags, edge types or properties.
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md:3]: where is the `create index` syntax?

"to add an index for existing properties on a tag or an edge type"?

File: docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md [hunk 2]
```diff
@@ -1,27 +1,56 @@
...
+
+Most graph queries start the traversal from a list of vertices or edges that are identified by their properties. Indexes make these global retrieval operations efficient on large graphs.
+
+## Prerequisites
+
+Before you create an index, make sure that the relative tag or edge type is created. For how to create tags or edge types, see CREATE EDGE (doc TODO) and CREATE EDGE (doc TODO).
+
+## Must-read for using index
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md:11]: unfortunately, you have to state very clearly that: 

consider which index to use:

1. text-based index (by elastic search)
2. native-index (by Nebula storage)

image

1. text-based index (by elastic search)

It is designed to full-text search (i.e., string and fix_string type of property in Nebula Graph). And can only be used for the following four searches: prefix, wildcard, regexp, fuzzy.

In this case, you have to **deploy an elastic search cluster** and introduce it to the Nebula Graph.

See xxx-link

2. native-index (by Nebula storage)

It is (almost) the same as the Nebula Graph v1.X index. see hereafter.

Comment (whitewum) [reply docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md:11]: re-draw this picture to cover this two concerpt.

**Review by whitewum — CHANGES_REQUESTED** [review]
let's rename to native-index and go on.
Explain the differences ( with text-based-index) and how-to choose in this /14.native-index-statement/

File: docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md (hunk 1 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md:3]: Now it's not only for properties. We can add an index for tags or edge types. But for is better than to. Thank you.

File: docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md (hunk 2 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md:11]: Thank you. I'll do this in another PR. This PR will focus on the `CREATE INDEX` topic.

**Review by whitewum — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**